src/components/Upgrade/content.js

```diff
@@ -162,3 +162,33 @@ export const highlights = [
     },
   },
 ]
+
+highlights['0.6'] = [
+  {},
+  {},
+  {},
+  {},
+  {},
+  {},
+  {
+    title: {
+      small: null,
+      large: 'What else is new?',
+    },
+    description: {
+      small: null,
+      large: `
+          A mobile-friendly UI, custom labels for apps
+          and addresses, an activity panel that keeps you
+          informed about the status of pending
+          transactions and much more!
+        `,
+    },
+    upgrade: {
+      small: 'Upgrade to 0.8 Camino',
+      large: 'Upgrade to 0.8 Camino',
+    },
+  },
+]
+
+export { highlights }
```

Comment:
Maybe we can change the export so that `highlights` becomes:

```js
highlights: {
  'latest': [...],
  '0.6': [...],
}
```

And then consumers can pick which one they need